### PR TITLE
12167 

### PR DESCRIPTION
**Pull request by vipullakhani**
Ticket #12167 Change the ZPLSC cabled driver to only produce time series data (and not echogram plot).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ https://conda.io/miniconda.html
 
 ## Create conda env
 
+    cd into mi-instrument and then type below command.
```
Comment (danmergens) [inline README.md]: `cd into mi-instrument and then type the command below:`

File: mi/instrument/kut/ek60/ooicore/driver.py [hunk 2]
```diff
@@ -545,41 +550,40 @@ def __init__(self, prompts, newline, driver_event):
 
         self._chunker = StringChunker(self.sieve_function)
 
-        log.info('Creating echogram processing pool with %d workers', POOL_SIZE)
-        self._process_echograms = True
-        self._pending_echograms = deque()
+        log.info('processing particles with processing pool %d workers', POOL_SIZE)
```
Comment (danmergens) [inline mi/instrument/kut/ek60/ooicore/driver.py]: `processing particles with %d workers`

File: mi/instrument/kut/ek60/ooicore/test/test_driver.py [hunk 3]
```diff
@@ -631,7 +651,7 @@ def test_got_data(self):
         self.push_data_to_driver(driver, self._test_file_notice)
         self.clear_data_particle_queue()
 
-        self.assert_particle_published_async(self.assert_file_data, True)
+        self.assert_particle_published_async(self.assert_file_data, False)
```
Comment (renegelinas) [inline mi/instrument/kut/ek60/ooicore/test/test_driver.py]: Why is this changed to False?

File: mi/instrument/kut/ek60/ooicore/zplsc_b.py [hunk 4]
```diff
@@ -185,7 +228,7 @@ def _build_parsed_values(self):
 
 def append_metadata(metadata, file_time, file_path, channel, sample_data):
     metadata[ZplscBParticleKey.FILE_TIME] = file_time
-    metadata[ZplscBParticleKey.ECHOGRAM_PATH]= file_path
+    metadata[ZplscBParticleKey.ECHOGRAM_PATH] = file_path
```
Comment (renegelinas) [inline mi/instrument/kut/ek60/ooicore/zplsc_b.py]: Is this still needed, since you are removing echogram creation?

File: mi/instrument/kut/ek60/ooicore/driver.py [hunk 5]
```diff
@@ -597,19 +601,38 @@ def echogram_thread(self):
...
                                                                     preferred_timestamp=DataParticleKey.INTERNAL_TIMESTAMP)
-                            parsed_sample = particle.generate()
+                            parsed_sample = metadata_particle.generate()
 
                             if self._driver_event:
                                 self._driver_event(DriverAsyncEvent.SAMPLE, parsed_sample)
 
+                            for counter, data_timestamp in enumerate(data_times):
+                                zp_data = {
+                                    ZplscBParticleKey.FREQ_CHAN_1: frequencies[1],
```
Comment (vipullakhani) [inline mi/instrument/kut/ek60/ooicore/driver.py]: it is 1 ,2  and 3

File: mi/instrument/kut/ek60/ooicore/driver.py [hunk 6]
```diff
@@ -597,19 +601,38 @@ def echogram_thread(self):
...
+                                    ZplscBParticleKey.VALS_CHAN_1: list(power_data_dict[1][counter]),
+                                    ZplscBParticleKey.FREQ_CHAN_2: frequencies[2],
+                                    ZplscBParticleKey.VALS_CHAN_2: list(power_data_dict[2][counter]),
+                                    ZplscBParticleKey.FREQ_CHAN_3: frequencies[3],
+                                    ZplscBParticleKey.VALS_CHAN_3: list(power_data_dict[3][counter]),
+                                }
+
+                                sample_particle = ZplscBSampleDataParticle(zp_data, port_timestamp=timestamp,
+                                                                    internal_timestamp=data_timestamp,
+                                                                    preferred_timestamp=DataParticleKey.INTERNAL_TIMESTAMP)
```
Comment (vipullakhani) [inline mi/instrument/kut/ek60/ooicore/driver.py]: For this cabled side, preferred timestamp is Internal timestamp because we will get timestamp for each slice of data.

File: mi/instrument/kut/ek60/ooicore/zplsc_b.py (hunk 4 above)
Comment (vipullakhani) [inline mi/instrument/kut/ek60/ooicore/zplsc_b.py]: No, it is not needed, i will remove this , thanks
